src/signature.rs

```diff
@@ -425,22 +425,22 @@ mod tests {
 	use xml::reader::*;
 
 	#[test]
-	fn get_hostname_none_present() {
+	fn hostname_none_present() {
```

Comment:
Hopefully not bike shedding but I think we can leave the test names alone.  They're more descriptive as `get_hostname_none_present` IMO.